rewrite-gradle/src/main/java/org/openrewrite/gradle/ChangeDependencyArtifactId.java

```diff
@@ -33,21 +34,21 @@
 @Value
 public class ChangeDependencyArtifactId extends Recipe {
     @Option(displayName = "Dependency pattern",
-            description = "A dependency pattern specifying which dependencies should have their artifactId updated. " +
-                    DependencyMatcher.STANDARD_OPTION_DESCRIPTION,
-            example = "com.fasterxml.jackson*:jackson-module*"
+        description = "A dependency pattern specifying which dependencies should have their artifactId updated. " +
+            DependencyMatcher.STANDARD_OPTION_DESCRIPTION,
+        example = "com.fasterxml.jackson*:jackson-module*"
     )
     String dependencyPattern;
 
     @Option(displayName = "New artifactId",
-            description = "The new artifactId to use.",
-            example = "rewrite-core")
+        description = "The new artifactId to use.",
+        example = "rewrite-core")
     String newArtifactId;
 
     @Option(displayName = "Dependency configuration",
-            description = "The dependency configuration to search for dependencies in.",
-            example = "api",
-            required = false)
+        description = "The dependency configuration to search for dependencies in.",
+        example = "api",
+        required = false)
     @Nullable
```

Comment:
Can someone check the autoformat intellij settings into source control? That would help greatly.

Reply:
They are the default intellij settings, which we adhere to so they don't need to be committed.

Reply:
Okay, so this is the default formatting, so this should theoretically be committed like this then

Reply:
can you check your Java tabs and indents?  I am going to guess your continuation is 4 and not 8.